spec/integ/matrix-client-event-emitter.spec.ts

```diff
@@ -92,9 +92,7 @@ describe("MatrixClient events", function () {
                                     type: "m.room.create",
                                     room: "!erufh:bar",
                                     user: "@foo:bar",
-                                    content: {
-                                        creator: "@foo:bar",
-                                    },
+                                    content: {},
```

Comment:
I've opted for removing the `creator` field from all test fixtures in an attempt to expose other places that might currently rely on it.

Reply:
This was addressed in the other thread.